pyhyp/pyHyp.py

```diff
@@ -212,10 +212,14 @@ def __init__(self, comm=None, options=None, commonOptions=None, debug=False, ski
             elif type(options) is dict:
                 # Only the root processor will print
                 if myid == 0:
-                    print("")
-                    print("")
-                    print("Running case %d : %s" % (index, options["inputFile"]))
-                    print("")
+                    # change the printout based on which option is actually present
+                    if "inputFile" in options.keys():
+                        caseName = options["inputFile"]
+                    elif "outputFile" in options.keys():
+                        caseName = options["outputFile"]
+                    else:
+                        caseName = ""
```

Comment:
Is there ever a case you dont have either `inputFile` or `outputFile` specified?

Reply:
I guess that's probably not going to happen, I adjusted the logic.